File: LeetCode/1512好数对的数目.cpp
int numIdenticalPairs(int* nums, int numsSize){
    int cnt = 0;
    for (int i = 0; i < numsSize; i++) {
        for (int j = i + 1; j < numsSize; j++) {
            if (nums[i] == nums[j]) cnt++;
        }
    }
    return cnt;
}

File: LeetCode/1512好数对的数目.rb
# @param {Integer[]} nums
# @return {Integer}
def num_identical_pairs(nums)
    ans = 0
    (0...nums.size - 1).each do |i|
        (i + 1...nums.size).each do |j|
            if nums[i] == nums[j]
                ans += 1
            end
        end
    end
    return ans
end